### ISSUE TITLE
Alpha value

### ISSUE THREAD

**Issue by theOyouknow** [piskelapp/piskel#826]
How hard would it be to enable an alpha value with `showAlpha` in spectrum?
Maybe an alpha value could only be accessed when there are multiple layers.

**Comment by realicraft** [comment]
Yes, but some people might want to be able to use the alpha value settings with only one layer. For instance, if they want to create a sprite that's partially transparent.

**Comment by kubsztal** [comment]
Duplicate of #434

**Comment by realicraft** [comment]
I don't think this is a duplicate. 434 is talking about the ability to use alpha colors in the color picker, while this one is talking about the alpha value per layer.

**Comment by kubsztal** [comment]
It says: "... enable an alpha value with showAlpha in spectrum". Spectrum is a color picker and layers have own opacity setting. Or am I missing something?
